app/celery/broadcast_message_tasks.py

```diff
@@ -111,6 +111,24 @@ def send_broadcast_event(broadcast_event_id):
         return
 
     broadcast_event = dao_get_broadcast_event_by_id(broadcast_event_id)
+
+    broadcast_message = broadcast_event.broadcast_message
+    # raise a P1 to alert team that broadcast is going out.
+    message = '\n'.join([
+        'Broadcast Sent',
+        '',
+        f'https://www.notifications.service.gov.uk/services/{broadcast_message.service_id}/current-alerts/{broadcast_message.id}',  # noqa
+        '',
+        f'This broacast has been sent on channel {broadcast_message.service.broadcast_channel}.',
+        f'This broadcast is targeted at areas {broadcast_message.areas.get("areas")}.',
+    ])
+    zendesk_client.create_ticket(
```

Comment:
what happens on preview/staging?

Reply:
we could possibly check by filling a feedback form on these environments?

Reply:
I think there is a way to create an incident through there, right?

Reply:
zendesk on preview/staging links to our normal zendesk.

i'll make this only run on prod.